frontend/src/app/homepage.cljs

```diff
@@ -102,11 +102,12 @@
 
 (defn render-stats []
   [:div {:id "progressbar"}
-   [:h5 "Are we there yet?"]
-   [:div {:id "progressbar-number"}
-    [:p (progress-width) "%"]]
    [:div {:id "progress"
-          :style {:width (str (progress-width) "%")}}]])
+          :style {:width (str (progress-width) "%")}}]
+   [:div
+    [:p "Collected " (backend-stats) " logs"]
```

Comment:
You want the number of total reports from the `backend-stats` therefore `(:total_reports @backend-stats)`